src/LoRa.cpp

```diff
@@ -66,17 +66,20 @@ int LoRaClass::begin(long frequency)
 {
   // setup pins
   pinMode(_ss, OUTPUT);
-  pinMode(_reset, OUTPUT);
-
-  // perform reset
-  digitalWrite(_reset, LOW);
-  delay(10);
-  digitalWrite(_reset, HIGH);
-  delay(10);
-
   // set SS high
   digitalWrite(_ss, HIGH);
 
+  if (_reset != -1)
+  {
```

Comment:
Could you please make this bracket consistent with the rest of the library.

Reply:
np - my stylecheck is really enemy with your file... I dont mind.